Automattic/vip-go-mu-plugins
# Sync Site Details when the siteurl/home changes

## Description
This PR improves the SDS syncing by adding a sync when the `siteurl/home changes`
This change should support both single and multisites.

## Changelog Description
### Improved Site Details sync

Small improvements in the site details sync to trigger a sync when the home/siteurl changes.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

*Single site*

1. Update the home/siteurl of a site `wp option update siteurl 'http://example.com' && wp option update home 'http://example.com'` 
1. Check that the SDS endpoint has been call and that the data has changed.

*Network site*

1. Update the home/siteurl of a site `wp option update siteurl 'http://example.com' --url=<your-old-url> && wp option update home 'http://example.com'  --url=<your-old-url>` 
1. Check that the SDS endpoint has been call and that the data has changed.
